### ISSUE TITLE
Add a11y to Swiper initialization object

### ISSUE THREAD

**Issue by brianjohnhanna** [stirling-brandworks/slab#42]
It looks like enabling a11y accessibility for the Swiper object is an opt-in setting. We should be setting to true for any examples included in Slab.

https://swiperjs.com/swiper-api#accessibility-a11y

It also appears that there is outstanding work to be done to ensure focusable elements in slides that aren't currently visible are not announced to screen readers. More information on that can be found here:

https://github.com/nolimits4web/swiper/issues/3149
